goatboard-api/src/main/kotlin/com/github/netbenix/managers/ConfigManager.kt

```diff
@@ -24,7 +24,7 @@ import java.io.File
 
 class ConfigManager private constructor() {
 
-    private val configPath = "/etc/goatboard/api.conf"
+    private val configPath = "C:\\Users\\Tomek\\Desktop\\goatboardconfig.cfg"
```

Comment:
Please revert to original
```suggestion
    private val configPath = "/etc/goatboard/api.conf"
```